docs/user_guide/install.md

```diff
@@ -18,7 +18,6 @@ Herbie is published on PyPI and you can install it with pip, _but_ it requires s
 
 - Python 3.9+
 - cURL
-- [cfgrib](https://github.com/ecmwf/cfgrib), which requires **_eccodes_** (most easily installed with conda)
```

Comment:
Let's keep this notice so not to surprise a user that **eccodes** is a requirement for cfgrib to work.